sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjb.java

```diff
@@ -358,6 +358,7 @@ public EventParticipantDto saveEventParticipant(@Valid EventParticipantDto dto,
 		validate(dto);
 
 		EventParticipant entity = fillOrBuildEntity(dto, existingParticipant, checkChangeDate);
+		entity.setArchived(event.isArchived());
```

Comment:
This will also set archived event participants of active events to active, even if they are supposed to stay active (which could happen quite frequently for events that have never been archived, but still contain archived event participants). Please make sure to move this logic to the logic that archives/dearchives events, i.e. when an event is archived, also archive all of its event participants.

Reply:
> Please make sure to move this logic to the logic that archives/dearchives events, i.e. when an event is archived, also archive all of its event participants.

The remark from @MateStrysewske does not match the expectation of the ticket to automatically archive an event participant when created on an archived event.
I think the `setArchived` should only be called when `existingParticipant == null´, and please add (or hopefully extend an existing unit test) to show that this behaviour is working as expected